lenovo/thinkpad/p14s/amd/gen2/default.nix

```diff
@@ -1,4 +1,8 @@
 { lib, pkgs, config, ... }:
+let
+  fwVersion = pkgs.linux-firmware.version;
+  oldLinuxFirmware = if (lib.hasPrefix "unstable" fwVersion) then (lib.versionOlder fwVersion "unstable-2023-02-10") else (lib.versionOlder fwVersion "20230210");
```

Comment:
I'm not sure which unstable version to match for exactly, but matching just `unstable` would probably already be enough. I'd be against removing the check. Even if it is no-longer supported, some people will still depend on 21.11 form some reason, and I'd hate to break their builds so soon.